### ISSUE TITLE
upgrade the store to have custom categories

### ISSUE THREAD

**Issue by dominater01** [hrydgard/ppsspp#11918]
i've figured out how the store works and have been adding a ton of stuff and i made my own categories like

-emulators-
emulator
-games-
game

etc

but having alot of downloads i would love if you implemented it so that

you know where the (back) letters are at the top left of the store?

maybe have it so you can choose a category at the top of the store by pressing right at the top

that way i dont have a huge list where you have to scroll for awhile because i plan on adding more stuff to my store

i noticed in the json file for yours you have a bunch of text for categories and stuff like that but to my knowledge they dont do anything

**Comment by ResurrectionRedmi9** [comment]
What is this?

**Comment by hrydgard** [comment]
Yeah, the category stuff I was intending to make use of if the store grew a lot. Unfortunately the store has kind of been stuck for a long time since I haven't had time or energy to prioritize it. Probably need to find a better way to manage it to let other people help, or something.
